pkg_resources/extern/__init__.py

```diff
@@ -1,6 +1,5 @@
 import sys
 
-
```

Comment:
Oops, I think this was a mistake. I think this is a PEP 8 violation. I'll clean it up.